Sources/Datadog/RUMMonitor.swift

```diff
@@ -248,53 +248,87 @@ public class RUMMonitor: DDRUMMonitor, RUMCommandSubscriber {
         )
     }
 
-    override public func startResourceLoading(resourceKey: String, url: URL, httpMethod: RUMHTTPMethod, attributes: [AttributeKey: AttributeValue]) {
```

Comment:
i think that having an extra `startResourceLoading(key:url:...)` method (in which `httpMethod` is default to `GET`) can be useful for some manual tracing cases.
how do you see the usage with `dataTask(withURL:)`?

Reply:
Good idea, done 👍. Now we have two methods for starting RUM Resource: one with `request`, another one with `url`.